lib/rules/no-obj-calls.js

```diff
@@ -11,6 +11,8 @@
 
 module.exports = {
     meta: {
+        type: "problem",
```

Comment:
What sort of bug are we warning users away from with this rule? (Wondering if this should be a "suggestion".)

Reply:
We're warning against people doing things like `Math()` and `JSON()`, which would cause a runtime error. As such, I think it should be a problem.